Demo/Demo/Models/VaultSessionID.swift

```diff
@@ -0,0 +1,7 @@
+import UIKit
+
+struct VaultSessionID: Codable {
+
+    let status: String?
+    let links: [LinkResponse]
```

Comment:
Nitpick - We can add related struct definitions to the same file to reduce the number of new files (similar to `CreateOrderParams.swift`)